packages/liquidation-sdk-viem/src/swap/1inch.ts

```diff
@@ -14,10 +14,15 @@ export namespace OneInch {
     apiKey = process.env.ONE_INCH_SWAP_API_KEY,
   ) {
     const url = new URL(getSwapApiPath(swapParams.chainId), API_BASE_URL);
-    url.searchParams.set("slippage", BigInt(swapParams.slippage).toString(16));
```

Comment:
This line did nothing, as `"slippage"` would be overwritten in the following code block. This is fortunate, as hexadecimal slippage doesn't seem compatible with the [1inch API spec](https://portal.1inch.dev/documentation/apis/swap/classic-swap/Parameter%20Descriptions/swap_params).